test/ts/migrations/2_deploy_dharma_contracts_test.ts

```diff
@@ -0,0 +1,113 @@
+import * as chai from "chai";
+import * as Units from "../test_utils/units";
+import { BigNumber } from "bignumber.js";
+import * as Web3 from "web3";
+import * as _ from "lodash";
+
+import { BigNumberSetup } from "../test_utils/bignumber_setup";
+import ChaiSetup from "../test_utils/chai_setup";
+
+// Set up Chai
+ChaiSetup.configure();
+const expect = chai.expect;
+
+// Configure BigNumber exponentiation
+BigNumberSetup.configure();
+
+// Dharma Contracts
+import { MultiSigWalletContract } from "../../../types/generated/multi_sig_wallet";
+import { DebtTokenContract } from "../../../types/generated/debt_token";
+import { DebtRegistryContract } from "../../../types/generated/debt_registry";
+import { RepaymentRouterContract } from "../../../types/generated/repayment_router";
+import { TokenTransferProxyContract } from "../../../types/generated/token_transfer_proxy";
+import { DebtKernelContract } from "../../../types/generated/debt_kernel";
+
+contract("Migration #2: Deploying Dharma Contracts", async (ACCOUNTS) => {
+    const CONTRACT_OWNER = ACCOUNTS[0];
+    const TX_DEFAULTS = { from: CONTRACT_OWNER, gas: 4000000 };
+
+    let wallet: MultiSigWalletContract;
+    let debtToken: DebtTokenContract;
+    let debtRegistry: DebtRegistryContract;
+    let tokenTransferProxy: TokenTransferProxyContract;
+    let repaymentRouter: RepaymentRouterContract;
+    let debtKernel: DebtKernelContract;
+
+    /*
+    Note(kayvon): we do make an implicit assumption here that the most recently
+    deployed versions of our Dharma contracts are in fact the ones deployed
+    during our migration process. These tests will break if and when we make
+    changes to how we configure our test environment.
+    */
+    describe("Deployment", () => {
+        it("should deploy the `MultiSigWallet` contract to the current network", async () => {
+            wallet = await MultiSigWalletContract.deployed(web3, TX_DEFAULTS);
+            expect(wallet.address).to.not.be.null;
```

Comment:
We can actually take these tests further and explicitly make sure that there's a contract deployed at `wallet.address`.

Please take a look at the `web3_utils#doesContractExistAtAddressAsync` function in the `dharma.js` repo -- you can use that code in the context of this test to test that there is _actually_ a contract deployed at each of these addresses.